README.md

```diff
@@ -75,25 +75,33 @@ domain = Domain(
 ```
 
 You can also use one of the many benchmarks available in BoFire.
-Here, we use the Himmelblau benchmark to demonstrate the ask/tell interface for
-proposing new experiments.
+Here, we use the Detergent benchmark to demonstrate the ask/tell interface for
+proposing new experiments with multi-objective Bayesian optimization.
 
 ```python
-from bofire.benchmarks.single import Himmelblau
-
-benchmark = Himmelblau()
-samples = benchmark.domain.inputs.sample(10)
-experiments = benchmark.f(samples, return_complete=True)
-
-from bofire.data_models.strategies.api import SoboStrategy
-from bofire.data_models.acquisition_functions.api import qNEI
 import bofire.strategies.api as strategies
-sobo_strategy_data_model = SoboStrategy(domain=benchmark.domain, acquisition_function=qNEI())
-
-sobo_strategy = strategies.map(sobo_strategy_data_model)
-
-sobo_strategy.tell(experiments=experiments)
-sobo_strategy.ask(candidate_count=1)
+from bofire.benchmarks.detergent import Detergent
+from bofire.data_models.strategies.api import QnehviStrategy, RandomStrategy
+
+# create benchmark
+detergent = Detergent()
+domain = detergent.domain
+
+# create initial data
+sampler = strategies.map(RandomStrategy(domain=domain))
```

Comment:
Add a comment here, something like:
`# use a random strategy to sample while satisfying constraints`

Reply:
done